.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
   # Install dependencies
   - conda create -n test-environment python=$TRAVIS_PYTHON_VERSION
   - source activate test-environment
-  - conda install dask numba numpy pandas pillow pytest toolz xarray
+  - conda install dask numba numpy pandas pillow pytest toolz xarray matplotlib
```

Comment:
Can we make a distinction here, at least via comments, between things we install that are actual dependencies (and thus should match pip and conda's list of dependencies) and things that are optional and are only for testing optional functionality?

It would be even better if there could be different travis targets with different sets of dependencies involved, so that we could verify that our optional dependencies are in fact optional, which is a trap that people often fall into, but I'm not sure how to do that in Travis.